dalle2_pytorch/dalle2_pytorch.py

```diff
@@ -72,6 +72,50 @@ def pad_tuple_to_length(t, length, fillvalue = None):
         return t
     return (*t, *((fillvalue,) * remain_length))
 
+# Diffusion prior model loading and saving helper functions
+
+def load_diffusion_model(dprior_path, image_embed_dim, device, wandb_entity, wandb_project):
```

Comment:
can you remove the wandb params and move image_embed_dim param in the checkpoint ?

Reply:
Ok, will do .